tests/integration/ceph_csi_test.go

```diff
@@ -36,18 +39,30 @@ const (
 	csiTestCephFSPodName = "csi-test-cephfs"
 )
 
-func runCephCSIE2ETest(helper *clients.TestClient, k8sh *utils.K8sHelper, s suite.Suite, namespace string) {
-	if isCSIRunnig(k8sh, namespace) {
+func runCephCSIE2ETest(helper *clients.TestClient, k8sh *utils.K8sHelper, s suite.Suite, t *testing.T, namespace string) {
+	if isCSIRunnig(k8sh, t, namespace) {
 		logger.Info("test Ceph CSI driver")
 		createCephCSISecret(helper, k8sh, s, namespace)
 		createCephPools(helper, s, namespace)
 		createCSIStorageClass(k8sh, s, namespace)
-		createCSIRBDTestPod(k8sh, s, namespace)
-		createCSICephFSTestPod(k8sh, s, namespace)
+		createAndDeleteCSIRBDTestPod(k8sh, s, namespace)
+		createAndDeleteCSICephFSTestPod(k8sh, s, namespace)
+
+		//cleanup resources created
+		deleteCephCSISecret(k8sh, namespace)
+		deleteCephPools(helper, namespace)
+		deleteCSIStorageClass(k8sh, namespace)
+
 	}
 }
 
-func isCSIRunnig(k8sh *utils.K8sHelper, namespace string) bool {
+func isCSIRunnig(k8sh *utils.K8sHelper, t *testing.T, namespace string) bool {
```

Comment:
`isCSIRunning()` just doesn't seem to be working anymore. For example, if you look in [this log](https://jenkins.rook.io/blue/rest/organizations/jenkins/pipelines/rook/pipelines/rook/branches/PR-3520/runs/4/nodes/55/steps/114/log/?start=0) and search for `TestBlockCSI_SmokeTest`, you will see that we are waiting indefinitely for the csi plugins to start. Perhaps the csi pod labels changed? In any case, I think we can just get rid of this `isCSIRunning()` method (let's not wait for the csi pods, they should be running anyway) and just implement the call to `!k8sh.VersionAtLeast("1.13.0")` from the `runCephCSIE2ETest()` method.

Note that the test log shows that we're failing to wait for the csi pods:
```
2019-07-26 15:03:34.685374 E | integrationTest: rbd pods are not running,failed with error Giving up waiting for pod with label app=csi-rbdplugin in namespace smoke-ns-system to be running
```

Reply:
this is failing due to the issue mentioned in #3324 (comment)